integreat_cms/cms/templates/linkcheck/link_list_row.html

```diff
@@ -103,7 +103,7 @@
             <a title="{% translate "Replace URL globally" %}"
                href="{% url 'edit_url' url_id=url.id url_filter=view.kwargs.url_filter region_slug=request.region.slug %}{{ pagination_params }}#replace-url"
                class="btn-icon">
-                <i icon-name="edit"></i>
+                <i icon-name="pencil"></i>
```

Comment:
```suggestion
                <i icon-name="pen-square"></i>
```
`edit` was renamed to `pen-square`, see: https://github.com/lucide-icons/lucide/pull/1286/files#diff-bdfb530ef0c966a6a14707783cd0f1bb179f852bb6b24e528d67028c6eda28c8